spec/fixtures/pacman-dsl.rb

```diff
@@ -0,0 +1,11 @@
+svg.setup width: "140", height: "100", style: "background:#ddd"
+
+svg.build do
```

Comment:
Would it be nicer without the `svg` prefix here?

Reply:
Agree. Would be much nicer. There is one use case to keep `svg`, and it is to allow that single file to merge multiple SVGs (as long as the final result is still stored in the `svg` object). 

Do you prefer to drop support for this use case, in favor of removing the `svg` prefix, and making it more DSLish?

Reply:
@gingermusketeer see #7